platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegionMetadata.java

```diff
@@ -43,21 +37,4 @@ public String toString() {
         return "OfflineRegionMetadata{metadata=" + metadata + "}";
     }
 
-    /*
-     * byte[] utils
-     */
-
-    public static byte[] serialize(Object obj) throws IOException {
```

Comment:
Since we're removing these utility methods, I think we should revisit https://github.com/mapbox/mapbox-gl-native/pull/4085#discussion_r53835207 and consider just using `byte[] OfflineRegion::getMetadata()`.

The TestApp code is probably a good example. The double `.getMetadata().getMetadata()` looks pretty awkward to me.

```
String json = new String(offlineRegion.getMetadata().getMetadata(), CustomMetadata.CHARSET);
```

@zugaldia @tobrun WDYT? 

I think this is better:

```
String json = new String(offlineRegion.getMetadata(), CustomMetadata.CHARSET);
```